README.md

```diff
@@ -31,6 +31,8 @@ There are 2 extra functions for special lookups in the Valhalla database (for cu
 
 ## Demo Access
 
+> Important: The demo API key is not available and demo access is not possible anymore. Hence, the following section and everything regarding the given demo API key is outdated.
```

Comment:
Isn't this too broad?

The demo key is still very much supported and working; only the access to the rule info endpoint has been disabled for the demo key.

Same for changes below.